pkg/querier/distributor_queryable.go

```diff
@@ -165,7 +182,14 @@ func (q *distributorQuerier) streamingSelect(ctx context.Context, sortSeries boo
 		return storage.EmptySeriesSet()
 	}
 
-	return series.NewConcreteSeriesSet(sortSeries, serieses)
+	seriesSet := series.NewConcreteSeriesSet(sortSeries, serieses)
+
+	if returnPartialData {
+		warning := annotations.Annotations(nil)
```

Comment:
Is removing other warnings from the seriesSet intentional?

Reply:
Oops, didn't realize seriesSet may contain warnings. Will fix it.

Reply:
Fixed